src/csgcca.c

```diff
@@ -37,6 +37,8 @@ const char *wrapper_debug_envvar_name = "DEBUG_CSGCCA";
 
 const char *analyzer_name = "gcc";
 
+const char *analyzer_bin_envvar = "CSGCCA_ANALYZER_BIN";
```

Comment:
Could you please document the variable here?

https://github.com/csutils/cscppc/blob/f02ccc233408a3b8cbc9b31989a459acfacdf567/doc/csgcca.adoc#environment-variables